code/src/Form.js

```diff
@@ -0,0 +1,101 @@
+import React, { useState } from "react";
+import Question1 from "./Question1";
+import Question2 from "./Question2";
+import Question3 from "./Question3";
+import Question4 from "./Question4";
+import Summary from "./Summary";
+
+const Form = () => {
+  const [name, setName] = useState("");
+  const [city, setCity] = useState("");
+  const [training, setTraining] = useState([]);
+  const [experience, setExperience] = useState();
+  const [question, setQuestion] = useState("Question1");
+
+  const questionList = [
+    "Question1",
+    "Question2",
+    "Question3",
+    "Question4",
+    "Summary",
+  ];
+
+  const handleSliderChange = (newValue) => {
+    setExperience(newValue);
+  };
+
+  const handleNameChange = (event) => {
+    setName(event.target.value);
+  };
+
+  const handleCityChange = (event) => {
+    setCity(event.target.value);
+  };
+
+  const handleTrainingChange = (trainingValue) => {
+    training.includes(trainingValue)
+      ? setTraining(training.filter((item) => item !== trainingValue))
+      : setTraining([...training, trainingValue]);
+  };
+
+  const handleNextQuestion = (event) => {
+    event.preventDefault();
+    const nextQuestion = questionList.indexOf(question);
+    setQuestion(questionList[nextQuestion + 1]);
+  };
+
+  return (
+    <>
+      {question !== "Summary" ? (
```

Comment:
Great use of ternary operator here but maybe keep it consistent and do it like you do on the other ones? "&&"